handler/src/main/java/io/netty/handler/ssl/JdkBaseApplicationProtocolNegotiator.java

```diff
@@ -168,40 +168,40 @@ public String noSelectMatchFound() throws Exception {
         }
     }
 
-    protected static class NoFailProtocolSelectionListener implements ProtocolSelectionListener {
-        private final JdkSslEngine jettyWrapper;
+    private static class NoFailProtocolSelectionListener implements ProtocolSelectionListener {
+        private final JdkSslEngine engineWrapper;
         private final List<String> supportedProtocols;
 
-        public NoFailProtocolSelectionListener(JdkSslEngine jettyWrapper, List<String> supportedProtocols) {
-            this.jettyWrapper = jettyWrapper;
+        NoFailProtocolSelectionListener(JdkSslEngine engineWrapper, List<String> supportedProtocols) {
+            this.engineWrapper = engineWrapper;
             this.supportedProtocols = supportedProtocols;
         }
 
         @Override
         public void unsupported() {
-            jettyWrapper.getSession().setApplicationProtocol(null);
+            engineWrapper.getSession().setApplicationProtocol(null);
         }
 
         @Override
         public void selected(String protocol) throws Exception {
             if (supportedProtocols.contains(protocol)) {
-                jettyWrapper.getSession().setApplicationProtocol(protocol);
+                engineWrapper.getSession().setApplicationProtocol(protocol);
             } else {
                 noSelectedMatchFound(protocol);
             }
         }
 
-        public void noSelectedMatchFound(String protocol) throws Exception {
+        protected void noSelectedMatchFound(String protocol) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)